cache-url/package.json

```diff
@@ -23,13 +26,19 @@
     "url": "https://github.com/ampproject/amp-toolbox/issues"
   },
   "homepage": "https://github.com/ampproject/amp-toolbox#readme",
-  "devDependencies": {
-    "eslint": "5.6.1",
-    "eslint-config-google": "0.10.0",
-    "jasmine": "3.2.0"
-  },
   "dependencies": {
     "mime-types": "^2.1.19",
```

Comment:
I think we can remove this dependency now

Reply:
Done with the master merge 👍